.github/workflows/postgresql.yml

```
@@ -0,0 +1,118 @@
+          - 5432:5432
+        env:
+          CRATE_HEAP_SIZE: 4g
+      postgresql:
+        image: postgres:${{ matrix.postgresql-version }}
+        ports:
+          - 5433:5432
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
+
+    name: "
+    Python ${{ matrix.python-version }}, 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)